src/app/pages/services/components/service-webdav/service-webdav.component.html

```diff
@@ -0,0 +1,68 @@
+<mat-card class="form-card">
+  <mat-card-content>
+    <form [formGroup]="form" class="ix-form-container" (submit)="onSubmit()">
+      <ix-fieldset [title]="helptext.fieldset_title">
+        <ix-select
+          [formControlName]="protocol.fcName"
+          [label]="protocol.label"
+          [tooltip]="protocol.tooltip"
+          [options]="protocol.options"
+          (ngModelChange)="onProtocolChange($event)"
```

Comment:
I think (ngModelChange) [should not be used](https://next.angular.io/api/forms/FormControlName#use-with-ngmodel-is-deprecated) with formControlName.
Try using `this.form.controls['fcName'].valueChanges` in component.